opencensus/stats/stats.py

```diff
@@ -14,12 +14,11 @@
 
 from datetime import datetime
 
-from opencensus.metrics.export.metric_producer import MetricProducer
 from opencensus.stats.stats_recorder import StatsRecorder
 from opencensus.stats.view_manager import ViewManager
 
 
-class _Stats(MetricProducer):
+class _Stats(object):
```

Comment:
We still need stats to be a metrics producer until deprecating stats completely.